Jenkinsfile

```diff
@@ -100,6 +100,13 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
                 utils.shwrap("""
                 coreos-assembler buildprep s3://${s3_builddir}
                 """)
+            } else if (!prod && utils.path_exists("/srv/devel/${devel_prefix}/build")) {
+                // XXX: should integrate this into buildprep
+                utils.shwrap("""
+                # really we only need some of the artifacts, but meh...
```

Comment:
yeah I thought we only really needed the commit object and maybe the meta.json or something